atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -949,17 +949,14 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
         for (Map.Entry<Cell, Value> e : rawResults.entrySet()) {
             bytes += e.getValue().getContents().length + Cells.getApproxSizeOfCell(e.getKey());
         }
-        if (bytes > TransactionConstants.ERROR_LEVEL_FOR_QUEUED_BYTES
-                && !AtlasDbConstants.TABLES_KNOWN_TO_BE_POORLY_DESIGNED.contains(tableRef)) {
-            log.error("A single get had a lot of bytes: {} for table {}. The number of results was {}. "
-                            + "The first 10 results were {}. This can potentially cause out-of-memory errors.",
-                    bytes, tableRef.getQualifiedName(), rawResults.size(), Iterables.limit(rawResults.entrySet(), 10),
-                    new RuntimeException("This exception and stack trace are provided for debugging purposes."));
-        } else if (bytes > TransactionConstants.WARN_LEVEL_FOR_QUEUED_BYTES && log.isWarnEnabled()) {
+        if (bytes > TransactionConstants.WARN_LEVEL_FOR_QUEUED_BYTES && log.isWarnEnabled()) {
             log.warn("A single get had quite a few bytes: {} for table {}. The number of results was {}. "
-                            + "The first 10 results were {}.",
-                    bytes, tableRef.getQualifiedName(), rawResults.size(), Iterables.limit(rawResults.entrySet(), 10),
-                    new RuntimeException("This exception and stack trace are provided for debugging purposes."));
+                    + " Enable debug logging for more information.",
```

Comment:
note: We'll have two spaces between the sentences this way - I'd suggest removing the space at the beginning of this line if only 1 was intended. I'm fine either way.